rrmngmnt/storage.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a {active} {vg_name}/{lv_name}'
+PVSCAN_CACHE_CMD = 'pvscan --cache'
+PVSCAN_CMD = 'pvscan'
+FIND_CMD = 'find / -name %s'
+CREATE_FILE_CMD = 'touch %s/%s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to local directory
+
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
+        temporary folder is created and it is the responsible of the
+        caller to remove it at the end
+        :type target
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'defaults']
+        :return: Path to mount point if succeeded, None otherwise
+        """
+        if target is None:
+            target = '/tmp/mnt_point'
+        cmd = ['mkdir', target]
+        self.host.executor().run_cmd(cmd)
+
+        cmd = ['mount', source, target]
+        if opts:
+            cmd.extend(opts)
+        rc, out, err = self.host.executor().run_cmd(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to mount source %s to target %s. Output: %s",
+                source, target, out
+            )
+            return None
+        return target
+
+    def umount(self, mount_point):
+        """
+        Performs 'umount' on 'mount_point' directory and remove
+        the 'mount_point'
+
+        :param mount_point: Path to directory that should be un mounted
+        :type mount_point: str
+        :return: True if umount operation and mount point removal
+        succeeded, False otherwise
+        :rtype: bool
+        """
+        rc, out, err = self.host.executor().run_cmd(
+            ['umount', mount_point, '-f', '-v']
+        )
+        if rc:
+            self.logger.error(
+                "failed to umount directory: %s, output: %s, error: %s",
+                mount_point, out, err
+            )
+            return False
+        rc = self.host.fs.rmdir(mount_point)
```

Comment:
what if I don't want to remove the directory, remember that this resource is not related to our tests anymore. I think that it is better to get this as parameter